packages/moonstone/ExpandableItem/ExpandableItem.js

```diff
@@ -271,7 +271,7 @@ const ExpandableItemBase = kind({
 	},
 
 	computed: {
-		className: ({open, styler}) => (styler.append({open})),
+		className: ({disabled, open, styler}) => disabled ? null : (styler.append({open})),
```

Comment:
Returning `null` here will overwrite both the `kind`-managed and author-provided classes. We should continue to use `styler` but conditionally apply `open` based on the combined state of `open` and `disabled` as we do in the computed `open` prop below.